docs/zh_CN/API.md

```diff
@@ -0,0 +1,1450 @@
+|        | ``InternalException`` : 内部错误。        |
+
+
+__示例__
+
+
+单个对象的最大大小限制在5TB。putObject在对象大于5MiB时，自动使用multiple parts方式上传。这就当上传失败时，客户端只需要上传未成功的部分即可（类似断点上传）。上传的对象使用MD5SUM签名进行完整性验证。
+
+```java
+try {
+  StringBuilder builder = new StringBuilder();
+  for (int i = 0; i < 1000; i++) {
+    builder.append("Sphinx of black quartz, judge my vow: Used by Adobe InDesign to display font samples. ");
+    builder.append("(29 letters)\n");
+    builder.append("Jackdaws love my big sphinx of quartz: Similarly, used by Windows XP for some fonts. ");
+    builder.append("(31 letters)\n");
+    builder.append("Pack my box with five dozen liquor jugs: According to Wikipedia, this one is used on ");
+    builder.append("NASAs Space Shuttle. (32 letters)\n");
+    builder.append("The quick onyx goblin jumps over the lazy dwarf: Flavor text from an Unhinged Magic Card. ");
+    builder.append("(39 letters)\n");
+    builder.append("How razorback-jumping frogs can level six piqued gymnasts!: Not going to win any brevity ");
+    builder.append("awards at 49 letters long, but old-time Mac users may recognize it.\n");
+    builder.append("Cozy lummox gives smart squid who asks for job pen: A 41-letter tester sentence for Mac ");
+    builder.append("computers after System 7.\n");
+    builder.append("A few others we like: Amazingly few discotheques provide jukeboxes; Now fax quiz Jack! my ");
+    builder.append("brave ghost pled; Watch Jeopardy!, Alex Trebeks fun TV quiz game.\n");
+    builder.append("- --\n");
+  }
+  ByteArrayInputStream bais = new
+  ByteArrayInputStream(builder.toString().getBytes("UTF-8"));
+  // 创建对象
+  minioClient.putObject("mybucket", "myobject", bais, bais.available(), "application/octet-stream");
+  bais.close();
+  System.out.println("myobject is uploaded successfully");
+} catch(MinioException e) {
+  System.out.println("Error occurred: " + e);
+}
+```
+
+<a name="putObject"></a>
+### putObject(String bucketName, String objectName, String fileName)
+
+`public void putObject(String bucketName, String objectName, String fileName)`
+
+通过文件上传到对象中。
+[查看 Javadoc](http://minio.github.io/minio-java/io/minio/MinioClient.html#putObject-java.lang.String-java.lang.String-java.lang.String-)
+
+__参数__
+
+
+|参数   | 类型	  | 描述  |
+|:--- |:--- |:--- |
+| ``bucketName``  | _String_  | 存储桶名称。  |
+| ``objectName``  | _String_  | 存储桶里的对象名称。 |
+| ``fileName``  | _String_  | File name. |
+
+
+| 返回值类型	  | 异常   |
+|:--- |:--- |
+|  None  | 异常列表： |
+|        |  ``InvalidBucketNameException`` : 不合法的存储桶名称。 |
+|        | ``NoResponseException`` : 服务器无响应。            |
+|        | ``IOException`` : 连接异常。            |
+|        | ``org.xmlpull.v1.XmlPullParserException`` : 解析返回的XML异常。            |
+|        | ``ErrorResponseException`` : 执行失败异常。            |
+|        | ``InternalException`` : 内部错误。        |
+
+__示例__
+
+
+单个对象的最大大小限制在5TB。putObject在对象大于5MiB时，自动使用multiple parts方式上传。这就当上传失败时，客户端只需要上传未成功的部分即可（类似断点上传）。上传的对象使用MD5SUM签名进行完整性验证。
```

Comment:
Ditto

Reply:
There are two putObject methods with different paramsters.And yes, the descritions are the same in the  original  docs.I better just delete the duplicated descrition.